src/main/scala/com/cloudant/clouseau/IndexManagerService.scala

```diff
@@ -94,6 +95,19 @@ class IndexManagerService(ctx: ServiceContext[ConfigurationArgs]) extends Servic
   val openTimer = metrics.timer("opens")
   val lru = new LRU()
   val waiters = Map[String, List[(Pid, Any)]]()
+  val countLocksEnabled = ctx.args.config.getBoolean("clouseau.count_locks", false)
+  if (countLocksEnabled) {
+    val lockClass = Class.forName("org.apache.lucene.store.NativeFSLock")
+    val field = lockClass.getDeclaredField("LOCK_HELD")
+    field.setAccessible(true)
+    val LOCK_HELD = field.get(null).asInstanceOf[HashSet[String]]
+    field.setAccessible(false)
```

Comment:
can we still call `size` on it if you make the private field inaccessible again?

Reply:
Yes, I was able to access `size` after marking the field inaccessible.